core/src/main/resources/i18n/displayStrings.properties

```
@@ -2245,12 +2245,16 @@ dao.factsAndFigures.dashboard.avgPrice30=30 days average BSQ/BTC trade price
 dao.factsAndFigures.dashboard.marketCap=Market capitalisation (based on trade price)
 dao.factsAndFigures.dashboard.availableAmount=Total available BSQ
 
+dao.factsAndFigures.supply.issuedVsBurnt=BSQ issued v. burnt
```

```suggestion
dao.factsAndFigures.supply.issuedVsBurnt=BSQ issued v. BSQ burnt
```

I think this is better, but not a strong opinion.

---

I like the conciseness in the original and the clarity in the suggestion. Clarity > conciseness, I suppose.